docs: fix typo in `fakeTimers.shouldClearNativeTimers` option (#6743)
vitest-dev/vitest: docs/config/index.md

#### fakeTimers.shouldClearNativeTimers

- **Type:** `boolean`
- **Default:** `true`

> Tells fake timers to clear "native" (i.e. not fake) timers by delegating to their respective handlers. These are not cleared by default, leading to potentially unexpected behavior if timers existed prior to starting fake timers session.

Reviewer: Let's make a paragraph here consistent:

```suggestion
Tells fake timers to clear "native" (i.e. not fake) timers by delegating to their respective handlers. When disabled, it can lead to potentially unexpected behavior if timers existed prior to starting fake timers session.
```
